vital/set.h

```diff
@@ -95,8 +95,8 @@ class set
   virtual T at( size_t index ) = 0;
```

Comment:
Apologies for not noticing before, but... this overload set doesn't seem to make sense; there is not a practical difference between the `const` and non-`const` overloads.